app/controllers/publishers_controller.rb

```diff
@@ -113,7 +113,7 @@ def ensure_email
     publisher = Publisher.find(params[:id])
 
     if token && PublisherTokenAuthenticator.new(publisher: publisher, token: token).perform(consume: false)
-      @publisher = publisher
+      @publisher = publisher.confirm_pending_email!(params[:confirm_email])
```

Comment:
So it seems there are two endpoints. This endpoint is hit when you first set up the account, and the other endpoint is an hybrid login/email-change endpoint.

I'm just trying to restore the old behavior of confirming through that endpoint as well. But this may be step-up bypass on a second thought as well